app/views/hyku_addons/account_settings/settings/_boolean_field_content.html.erb

```diff
@@ -2,7 +2,7 @@
 <%=f.fields_for :settings do |ff| %>
   <%= ff.label field_name %><br>
   <%= ff.label t("settings.helptexts.#{field_name}") %><br>
-  <%= ff.check_box field_name, {checked: @account.settings[field_name]}, true, false %>
```

Comment:
Why did you decide to remove this?  It looks like this was setting the field in the form based upon the saved value, but now it looks like it will always show as unchecked even when it has be saved as true.

Reply:
This might be an error

Reply:
When these are set as Boolean's they are still saved as strings when I look at them in the rails console. I have kept them as Strings and will change the test